### ISSUE TITLE
3d basemaps for TDI Districts 

### ISSUE THREAD

**Issue by IanJakus** [Siqister/cartodb_test_app#28]
Add in 3d basemaps for all ten districts.

Remaining:
- [x] Revere
- [ ] Worcester
- [x] Brockton
- [x] Springfield
- [x] Haverhill
- [x] New Bedford
- [x] Peabody
- [x] Holyoke
- [ ] Pittsfield

**Comment by nupoormonani** [comment]
- [x] Change opacity of 3D buildings layer and move above parcels layer

**Comment by IanJakus** [comment]
I love the look of the buildings with the parcels underneath. However now I cannot select the parcels.

**Comment by allthesignals** [comment]
@IanJakus This is fixed

**Comment by allthesignals** [comment]
@nupoormonani Just pinging you on this - I don't have any of the unchecked 3d basemaps in the list above. I'm not sure what the rollout schedule was for this, but wanted to make sure I'm not missing anything.

**Comment by nupoormonani** [comment]
@allthesignals Sorry, didn't realize this slipped through the cracks. Attaching the rest of them here. I believe you also need coordinates for one of the points on the map? We can work on that when you are here soon. 
![2016-07-19 worcester zoom-02](https://cloud.githubusercontent.com/assets/19911108/16965904/35221c72-4dd0-11e6-98f4-46eaa5a3a899.jpg)
![2016-07-19 holyoke zoom-02](https://cloud.githubusercontent.com/assets/19911108/16965902/351fe8f8-4dd0-11e6-8353-8f638cf85d85.jpg)
![2016-07-19 pittsfield zoom-02](https://cloud.githubusercontent.com/assets/19911108/16965905/35258d58-4dd0-11e6-851c-6418a33990ac.jpg)
![2016-07-19 new bedford zoom-02](https://cloud.githubusercontent.com/assets/19911108/16965906/3528d6b6-4dd0-11e6-9c0c-44e50e91cc30.jpg)
![2016-07-19 peabody zoom_2](https://cloud.githubusercontent.com/assets/19911108/16965903/3520cf0c-4dd0-11e6-8d35-356ac3a14b48.jpg)

**Comment by allthesignals** [comment]
@nupoormonani Yes, these need to be pngs, too, with transparencies. I will need the bounding box coordinates of the output PNGs. Thanks!
